server/src/main/java/org/apache/iotdb/db/metadata/MManager.java

```diff
@@ -646,6 +648,7 @@ public boolean deleteStorageGroupsFromMTree(List<Path> deletePathList)
           IoTDBConfigDynamicAdapter.getInstance().addOrDeleteStorageGroup(-1);
           mgraph.deleteStorageGroup(delStorageGroup);
           seriesNumberInStorageGroups.remove(delStorageGroup);
+          ActiveTimeSeriesCounter.getInstance().delete(delStorageGroup);
```

Comment:
Do you need to updateActiveRatio here, or updateActiveRatio before the next use such as in `TsFileProcessor.getMemtableSizeThresholdBasedOnSeriesNum`?

Reply:
It would be unsafe for the memory control since the delete storage group's memtable may not be flushed yet.